velocity_pubtime: 0.1
turn_radius: 30.0
max_turnspeed: 15.0
max_speed: 1.5
